dbt/models/default/schema.yml

```diff
@@ -69,27 +69,11 @@ models:
     tests:
       # Unique by card and year
       - dbt_utils.unique_combination_of_columns:
-          name: vw_card_res_char_unique_by_card_and_year
+          name: vw_card_res_char_unique_by_pin_card_and_year
           combination_of_columns:
+            - pin
             - year
             - card
-          config:
-            error_if: ">913"
```

Comment:
Adding PIN makes this test pass for all values, which is nice!

Reply:
Awesome. Good thing to remember since I'm always less explicit/clear/helpful than I should be; card is essentially a sub-PIN, so we'll only ever work with card in tandem with PIN.

Reply:
Will these unit tests get tripped up as more rows are added to our tables? This is a distinct possibility due to what we call "divisions" - parcels getting split into more than one piece and new PINs being created. It happens on a pretty regular basis, unfortunately... we might want to choose a predefined time in the past for unit tests (2018, for a random example), if that's the case.

Reply:
Cool, just wanted to be sure.